{
  "name": "@hidoo/gulp-util-error-handler",
  "version": "0.6.2",
  "description": "Utility that handle task error for gulp.",
  "engines": {
    "node": ">=8.0.0"
  },
  "main": "lib/index.js",
  "files": [
    "lib/"
  ],
  "scripts": {
    "prebuild": "rimraf ./lib",
    "build": "babel ./src --out-dir ./lib",
    "test": "cross-env NODE_ENV=test mocha"
  },
  "keywords": [
    "gulp"
  ],
  "publishConfig": {
    "access": "public"
  },
  "author": "hidoo",
  "license": "MIT",
  "dependencies": {
    "chalk": "^2.4.1",
    "fancy-log": "^1.3.2"
  },
  "devDependencies": {
    "@babel/cli": "^7.1.5",
    "@babel/core": "^7.1.2",
    "@babel/preset-env": "^7.1.0",
    "@babel/register": "^7.0.0",
    "babel-preset-power-assert": "^3.0.0",
    "cross-env": "^5.2.0",
    "mocha": "^5.2.0",
    "power-assert": "^1.6.1",
    "rimraf": "^2.6.2",
    "sinon": "^6.1.5",
    "strip-ansi": "^5.0.0"
  }
}
